queries.sql

```diff
@@ -5,4 +5,36 @@ SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pickachu';
 SELECT name,escape_attempts FROM animals WHERE weight_Kg > 10.5;
 SELECT * FROM animals WHERE neutered='true';
 SELECT * FROM animals WHERE NOT name='Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-1-1';
+SAVEPOINT delete_one;
+UPDATE animals SET weight_kg = -1 * weight_kg;
+ROLLBACK TO delete_one;
+UPDATE animals SET weight_kg = -1 * weight_kg WHERE weight_kg < 0;
+COMMIT;
+SELECT COUNT(*) FROM animals;
+-- 9
+SELECT COUNT(*) FROM animals WHERE neutered = false;
+-- 3
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+-- 2
+SELECT AVG(weight_kg) FROM animals;
+-- 16.6488888888888889
+SELECT MAX(escape_attempts) FROM animals;
```

Comment:
This query answers the question  `Who escapes the most, neutered or not neutered animals?` but it's not fully correct because it missing the last condition(neutered or not neutered)

- [ ] Kindly update query